app/policies/share_policy.rb

```diff
@@ -9,11 +9,11 @@ def initialize(user, share)
   end
 
   def show?
-    user == share.user || user.member_of?(share.recipient)
+    user == share.user || (user.member_of?(share.recipient) && share.unrevoked?)
```

Comment:
this does bring up a bit of an interesting question actually - when the share is revoked, can the organisation no longer see the share at all, or can they just no longer see the document that's linked to it?

Reply:
I think this is a question for @workbygrant